index.html

```diff
@@ -1,697 +1,715 @@
+    }
+    .get-a-quote .tabs .tab.active {
+        background: #f4f8f9;
+        color: inherit;
+        border-bottom: 1px solid transparent;
+    }
+    .get-a-quote .tab-content {
+        display: none;
+    }
+    .get-a-quote .tab-content > * {
+        margin: 15px;
+    }
+    .get-a-quote .tab-content.active {
+        display: block;
+    }
+    .get-a-quote .tab-content label {
+        text-align: center;
+        display: block;
+    }
+    .get-a-quote .tab-content input {
+        text-align: right;
+        font-family: inherit;
+        font-weight: 200;
+        padding: 0.2em;
+        font-size: 1.2em;
+        width: 6em;
+    }
+    .get-a-quote .evaluation-text {
+        background: #cee8ef;
+        margin-left: 0;
+        margin-right: 0;
+        padding: 5px 15px;
+        font-size: 0.7em;
+        min-height: 4em;
+        transition: background 1.2s;
+    }
+    .get-a-quote .evaluation-text.good {
+        background: #c8efba;
+    }
+    .get-a-quote .evaluation-text.bad {
+        background: #efe6ba;
+    }
+    .get-a-quote .tab-content .button {
+        display: block;
+        width: 100%;
+        text-align: center;
+        background: #2c3e50;
+        border: none;
+        cursor: pointer;
+        text-decoration: none;
+        padding: 0.5em;
+        font-size: 1.4em;
+        color: white;
+        font-weight: 200;
+        font-family: inherit;
+        margin-left: 0;
+        margin-right: 0;
+        margin-bottom: 0;
+        transition: background-color 0.2s;
+    }
+    .get-a-quote .tab-content .button:hover {
+        background: #3f5e7c;
+    }
+    .get-a-quote .get-in-touch-explanation {
+        text-align: center;
+        font-size: 0.6em;
+    }
+</style>
+<section class="rest-sales">
+    <h1>Get a quote</h1>
+    <p>
+        We prefer working on a monthly budget.
+        We are experts in working in the most efficient way possible, playing to each others' strengths, making sure you get the most value delivered for your money.
+    </p>
+    <p>
+        If you want to hire us, see if we can make something happen within your budget here:
+    </p>
+    <section class="get-a-quote" id="get-a-quote">
+        <div class="header">
+            Get a quote
+        </div>
+        <div class="tabs">
+            <div class="tab active" data-tab="team">Team</div>
+            <div class="tab" data-tab="project">Project</div>
+            <div class="tab" data-tab="other">Other</div>
+        </div>
+        <div class="tab-content active" data-tab="team">
+            <p>
+                We can be your drop-in development team.
+                Let us know your budget, and we'll work something out!
+            </p>
+            <label class="input-container">
+                <input name="team-budget" placeholder="0,00"> DKK/month
+            </label>
+            <div class="evaluation-text">
+                Input a budget, and we'll work something out!
```

Comment:
very small font-size

Reply:
Hmm i find these comments on small font size hard to act on. Do you perceive a problem (are they actually hard to read at 100%? is focus on the wrong content, and should this be emphasized more by making it larger?) or are you just noting that the font size is smaller than that of other elements?

Reply:
(Because it's intentional that font size varies -- it's used to draw the eye to the important information, while leaving less important information there if people look around.)